### DISCUSSION TITLE
Ds18 for Floatyboard

### DISCUSSION THREAD

**Discussion by jangolen** [mp-se/gravitymon#157] [Ideas]
Hello.  Nice software, big thanks.  
Why not to add ds18 support for floaty board like a power monitor? 
I try to recompiled by myself, but have a several errors with library’s in platformio.  
2. Tilt emulation with Brewpiless needed some more time slot to browadcast BLE beacon messages. More time for broadcast, but less frequently.  Serial monitor on Brewpi is very helpful for tuning of timings.  It’s very useful not to hardcode this variable, but have it in web configuration.

**Comment by mp-se** [comment]
It would be possible to add support for a temp sensor, but you are probably better off just building an ispindel with an esp32c3 board instead. The floaty design is quite old and so is the board that is used. 

You can get the battery voltage with a small modification, see the docs.

Ive not created the ble part myself (its included from another project) but i can have a look to see what parameters could be tweaked. Its not a big deal to add more tuning options. Let me get back on this point.

**Comment by mp-se** [comment]
I guess its the active interval that you want to be able to configure or do you also see a need for broadcasting a few times?

**Reply by jangolen** [reply]
Hi, I'm now successfully compiled project with alternative ble broadcasting timings, but not yet find correct value for BrewPiLess. Brewpi has BLE scanning interval about 10 seconds.

**Reply by mp-se** [reply]
Then you probably should try to do the broadcast for +10 seconds in interval in order to see if it can read it. Just note that this will take a lot of battery power keeping the device running for that long. Why isnt brewpi doing BLE scanning in the background always ?
